test/onnx/verify_onnx.cpp

```diff
@@ -351,6 +351,60 @@ TEST_CASE(depthtospace_simple_test)
     EXPECT(migraphx::verify::verify_rms_range(result_vector, gold));
 }
 
+TEST_CASE(dynamicquantizelinear_1d_test)
```

Comment:
Suggestion: Please also add the first test listed in the reference:  where its zp is 153. Thanks.

https://github.com/onnx/onnx/blob/main/docs/Operators.md#dynamicquantizelinear